### PR TITLE
Integrate social icons on login page with symfony ux-icons

### LINKED ISSUE #36242: Integrate proper icons for facebook, twitter and github in the login page

### PR DESCRIPTION
**Pull request by mattgoud**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | install and configure [Symfony ux-icons](https://symfony.com/bundles/ux-icons/current/index.html) package, import github/facebook/x icons in new-theme, refacto Login page footer (add icons).
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Go to BO login page > see social medias icons on login page footer
| UI Tests          | https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/9385342113
| Fixed issue or discussion?     | Fixes #36242
| Related PRs       | If theme, autoupgrade or other module change is needed to make this change work, provide a link to related PRs here.
| Sponsor company   | PrestaShopCorp

### CODE REVIEW

**Comment by ps-jarvis** [comment]
Hi, thanks for this contribution!

Please consider opening an issue before submitting a Pull Request:
* If it's a bug fix, it helps maintainers verify that the bug is effectively due to a defect in the code, and that it hasn't been fixed already.
* It can help trigger a discussion about the best implementation path before a single line of code is written.
* It may lead the Core Product team to mark that issue as a priority, further attracting the maintainers' attention.

(Note: this is an automated message, but answering it will reach a real human)

File: app/config/admin/config.yml
Comment (jolelievre) [inline app/config/admin/config.yml]: You need to add a new line at the end of the file

File: src/PrestaShopBundle/Resources/views/Admin/Layout/login_layout.html.twig
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/views/Admin/Layout/login_layout.html.twig]: There was no text in the legacy page so you can remove those and only keep the icons

File: app/config/admin/config.yml
Comment (mattgoud) [reply app/config/admin/config.yml]: yes @jolelievre done

File: src/PrestaShopBundle/Resources/views/Admin/Layout/login_layout.html.twig
Comment (mattgoud) [reply src/PrestaShopBundle/Resources/views/Admin/Layout/login_layout.html.twig]: ok done

**Review by jolelievre — DISMISSED** [review]
Thanks @mattgoud

**Review by kpodemski — DISMISSED** [review]

**Review by jolelievre — APPROVED** [review]

**Comment by jolelievre** [comment]
@mattgoud I allowed myself to rebase your PR to fix a conflict that poped out after the merge PR

**Review by kpodemski — APPROVED** [review]

**Comment by jolelievre** [comment]
Thanks @mattgoud

**Pull request merged**
